### DISCUSSION TITLE
Airgap update to 2.3.2 Issue

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#1700] [General]
After mounting 2.3.2 iso to manager and running soup, to update soup, and running soup again to update the manager, is anyone else getting the error "/sbin/soup /sbin/soup: cannot execute binary file"? SO Landing page still states we are running 2.3.1

**Comment by TOoSmOotH** [comment]
Where are you running soup from?

**Reply by darth-drew** [reply]
/home/myUsername on the manager node.

**Reply by TOoSmOotH** [reply]
How are you mounting it?

**Comment by darth-drew** [comment]
I connect the new ISO to the manager cdrom drive via esxi gui, run sudo
soup, and the script prompts me to enter /dev/cdrom. It mounts the disc,
updates soup, and prompts me to run soup again, at which point I get the
error message.

On Wed, Oct 28, 2020 at 12:04 PM Mike Reeves <notifications@github.com>
wrote:

> How are you mounting it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1700#discussioncomment-114496>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADC5ZNTXIQTB3O6QI4DFJ4TSNA6IZANCNFSM4TCLNLJQ>
> .
>


-- 
Andrew R. Albertson
C: (540)908-8551

**Reply by TOoSmOotH** [reply]
Is the CD still mounted when you run the install the second time?

**Comment by darth-drew** [comment]
Yes sir.

On Wed, Oct 28, 2020 at 12:44 PM Mike Reeves <notifications@github.com>
wrote:

> Is the CD still mounted when you run the install the second time?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1700#discussioncomment-114540>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADC5ZNVZ2XKNGQWROCVTRODSNBC5BANCNFSM4TCLNLJQ>
> .
>


-- 
Andrew R. Albertson
C: (540)908-8551

**Reply by TOoSmOotH** [reply]
I can't reproduce. Can you do an md5sum on /usr/sbin/soup

**Reply by TOoSmOotH** [reply]
1a41b60e958d57b07c08744a8bb5f72e  /usr/sbin/soup

**Reply by TOoSmOotH** [reply]
One thing to try is to unmount the drive and try again if the soups match

**Comment by darth-drew** [comment]
md5sum does not match. I did try to unmount and try again but am getting
the same error message.

On Wed, Oct 28, 2020 at 2:46 PM Mike Reeves <notifications@github.com>
wrote:

> One thing to try is to unmount the drive and try again if the soups match
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1700#discussioncomment-114646>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADC5ZNVWYBDGTFLNPBJPZ6TSNBRJBANCNFSM4TCLNLJQ>
> .
>


-- 
Andrew R. Albertson
C: (540)908-8551

**Reply by TOoSmOotH** [reply]
you could mount the cd and then copy soup to `cp /cdmountdir/SecurityOnion/salt/common/tools/sbin/soup /opt/so/saltstack/default/salt/common/rools/sbin/` and then run `salt-call state.apply common`

Then unmount the CD and run soup

**Comment by darth-drew** [comment]
Sorry about this.. it's still saying  /sbin/soup /sbin/soup: cannot execute
binary file.. I can fresh install it's just a bit of a pain at this point.
Any other suggestions?

Thanks for your assistance.

On Thu, Oct 29, 2020 at 11:18 AM Mike Reeves <notifications@github.com>
wrote:

> you could mount the cd and then copy soup to cp
> /cdmountdir/SecurityOnion/salt/common/tools/sbin/soup
> /opt/so/saltstack/default/salt/common/rools/sbin/ and then run salt-call
> state.apply common
>
> Then unmount the CD and run soup
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1700#discussioncomment-115783>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADC5ZNUSK7SBWQXWXMZRPUDSNGBVZANCNFSM4TCLNLJQ>
> .
>


-- 
Andrew R. Albertson
C: (540)908-8551

**Reply by TOoSmOotH** [reply]
maybe try` sudo /usr/sbin/soup`

**Comment by darth-drew** [comment]
How strange. I reverted back to the same snapshot prior to upgrade I have two or three times by now. However, for whatever reason, this time sudo soup ran as expected both times.

**Reply by TOoSmOotH** [reply]
so you good?

**Comment by darth-drew** [comment]
Reverted back after the update bricked my docker images. Similar to another
thread. sudo /usr/sbin/soup gives the same error.


On Thu, Oct 29, 2020 at 12:51 PM Mike Reeves <notifications@github.com>
wrote:

> so you good?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1700#discussioncomment-115881>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADC5ZNVYHB7IWZ5MIJEUZQ3SNGMRVANCNFSM4TCLNLJQ>
> .
>


-- 
Andrew R. Albertson
C: (540)908-8551
